src/Pacman/Scores/index.js

```diff
@@ -2,19 +2,24 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './style.scss';
 
-export default function Scores({ score, lost, won }) {
+export default function Scores({ score, ended }) {
     let gameOver = null;
 
-    if (lost) {
-        gameOver = (
-            <span className="game-over">{'Game over!'}</span>
-        );
-    }
-
-    if (won) {
-        gameOver = (
-            <span className="game-over">{'Congratulations, you won!'}</span>
-        );
+    switch(ended) {
+        case 'WON': {
+            gameOver = (
+                <span className="game-over">{'Congratulations, you won!'}</span>
```

Comment:
This code is duplicated with line 17. The only difference is the message, which should be in a dictionary or something, e.g.

```js
const GAME_OVER_MESSAGE = {
  [WON]: 'Congratulations, you won!',
  [LOST]: 'Game over!',
};
```
Then
```js
{ended && <span className="game-over">{GAME_OVER_MESSAGE[ended]}</span>}
```